### ISSUE TITLE
[BUG] - Problems with immutable-Map 

### ISSUE THREAD

**Issue by lexon0011** [compodoc/compodoc#95]
Since the version "0.0.37" I get the following error: 

```
[10:13:32] Process page   : AppState
[10:13:32] TypeError: Cannot read property 'map' of undefined
```

I use map in the following way:

```
import { Iterable, Map, Record } from "immutable";

export interface IModelWithRecord {
    _record: Map<string, any>;
}
```

Transpiling the application with tsc and the same tsconfig.json works. 

Could you reproduce the error?

**Comment by vogloblinsky** [comment]
No.
Need the AppState code, here or by mail.
Regards

**Comment by lexon0011** [comment]
In the attachment you find the appState.ts and the two files which are included. If you need more input, please tell me!


Thanks!


Regards

Martin

________________________________
Von: Vincent Ogloblinsky <notifications@github.com>
Gesendet: Donnerstag, 26. Jänner 2017 10:27:36
An: compodoc/compodoc
Cc: Martin Schagerl; Author
Betreff: Re: [compodoc/compodoc] [BUG] - Problems with immutable-Map (#95)


No.
Need the AppState code, here or by mail.
Regards

-

**Comment by vogloblinsky** [comment]
Nope

**Comment by lexon0011** [comment]
Here are the files:

[AppState.zip](https://github.com/compodoc/compodoc/files/731993/AppState.zip)

**Comment by vogloblinsky** [comment]
Ok reproduced

**Comment by vogloblinsky** [comment]
Ok fixed, the support of decorators on properties is not supported right now.
New version this afternoon.

**Comment by lexon0011** [comment]
Thanks!

**Comment by vogloblinsky** [comment]
Fixed in 0.0.38
